src/client/routing/SecureRoute.js

```diff
@@ -16,9 +16,9 @@ export default function SecureRoute(props) {
 			path={path}
 			exact={exact}
 			render={(routerProps) => {
-				let roles = user.getIn(['data', 'roles']);
+				let roles = user.data.roles
 				// Only render the target page if the user has the required roles
-				if (requiredRoles.isSubset(roles) || roles.includes('admin')) {
+				if (requiredRoles.every((role) => roles.includes(role))) {
```

Comment:
Hmm, we may need to update the logic in every to also include the `role.includes('admin')`, something like:
```
if (roles.includes('admin') || requiredRoles.every((role) => roles.includes(role))) {
```

but let me know what you think. My though is that be default, we allow admins to see every page.  Currently if we configured the following routes:

```
{
  requiredRoles: [],
  name: 'Home'
},
{
  requiredRoles: ['manager'],
  name: 'Dashboard'
},
```

Everyone and admin could see Home, but only managers would see the Dashboard, the admin would not be able to see it unless he also had the manager role.

Reply:
Makes sense. I agree. I'll put that fix in thanks.